### ISSUE TITLE
engines do not respond to commands

### ISSUE THREAD

**Issue by YuraBz** [makerbase-mks/MKS-Robin#68]
Hi, I connected the driver drv8825 to the board, started running with the default configuration and driver once trying to scroll and no longer responds to the commands. checked various drivers and engines, the result is the same. In the application on the screen hang the inscription "error"

**Comment by YuraBz** [comment]
checked different vref, acceleration, speed, step, the result is the same. maybe someone has a working configuration file under xyz

**Comment by makerbase-mks** [comment]
The normal condition of the Robin board to work is to be able to check the temperature from the thermal port.

**Comment by YuraBz** [comment]
good day, With this I have taken care of. can one more question? 
І have mks slot 1.1 8pin and mks robin 2.4 with 9pin sd external port, can i connect them somehow?

**Comment by makerbase-mks** [comment]
No,the definition of pins are difference.You can replace it with mks slot2.

**Comment by YuraBz** [comment]
where can i find pinout or datasheet slot 1.1 & 2?
